lightcycle-api/build.gradle

```diff
@@ -9,10 +9,8 @@ checkstyle {
 
 dependencies {
   compileOnly deps.android
```

Comment:
what is this? deps.android?

Reply:
this is a trickiest part, there is a dependency to 
`android               : "com.google.android:android:4.1.1.4"` which is required for `ligthcycle-api` module since it has a basic android platform dependencies from Bundle and MenuItems and Intents. 
Let me know if you know how to replace this dependency or make it more clear.